low-level/files/opt/seagate/sspl/setup/sspl_setup.py

```diff
@@ -185,8 +185,8 @@ def validate(self):
         pass
 
     def process(self):
-        # TODO: Import relevant python script here for further execution.
-        pass
+        from files.opt.seagate.sspl.setup import sspl_setup_init
+        sspl_setup_init.Init(self.args).process
```

Comment:
You need to call two interfaces 

sspl_init.init(self.args) --> This will initialize and perform necessary validations. 
sspl_init.process() --> This will do actual initialzation. 

(please note interface names are small letters)

Reply:
agree. Also sspl_init.init(self.args) should be invoked from __init__() of this class InitCmd